### PR TITLE
Added TWNKL Token to Defaults

### PR DESCRIPTION
**Pull request by stephenhodgkiss**
Rainbow Currency Foundation is a Non-Profit Corporation, registered in Nevada
Business ID NV20171765116.

You simply hold a smart new cryptocurrency, that has been built to make global and local payment processes safer, cheaper, and easier. Purchases happen internally for this ... for example, it makes no sense for a customer wanting to buy a coffee to have to create a transaction on the blockchain for it ... this way payment is instant and has zero fees.

Thanks to a unique 2-wallet solution, the whole currency is always fully transparent, decentralized, and safe in the blockchain, while everyday payments for shopping and services can be made online and at POS fast, secure, and without fees.

To be qualified for online and local payments, the currency holder must go through a verification process to identify themselves, while only accredited business partners are allowed to receive these payments. Thus, we make it difficult and uncomfortable to participate for the bad guys.

The token is on a Standard ERC20 smart contract avoiding misuse. Contract Address: 0xfbd0d1c77b501796a35d86cf91d65d9778eee695

Rainbow Currency is NOT an investment into a certain project. When you own Rainbow Currency, you are not holding any partnership, stock, or revenue share rights.

There is never any expectation of profits, as it is not sold as an investment. Pure demand will generate an increase in token pricing and this will happen organically. Yes of course, we will have advertising campaigns and they will be managed by the company, as with any business, exposure is a good thing but it is used to promote products and services and never about the token.

### CODE REVIEW

File: tokens/tokens-eth.json [hunk 1]
```diff
@@ -419,7 +419,7 @@
 "symbol"      : "ALTS",
 "address"     : "0x638ac149ea8ef9a1286c41b977017aa7359e6cfa",
 "decimals"    : "18",
-"name"        : "ALTS Token",
+"name"        : "ALwnS Token",
```
Comment (SteveMieskoski) [inline tokens/tokens-eth.json]: Revert name to ALTS Token

File: tokens/tokens-eth.json [hunk 2]
```diff
@@ -8351,40 +8351,7 @@
...
-      "instagram" : "",
-      "linkedin"  : "",
-      "reddit"    : "",
-      "slack"     : "",
-      "telegram"  : "",
-      "twitter"   : "https://twitter.com/thenagaico",
-      "youtube"   : ""
-    }
-  },{
+},{
```
Comment (SteveMieskoski) [inline tokens/tokens-eth.json]: Revert removal of the NGC, NAGA Coin entry
`    {
    "symbol"      : "NGC",
    "address"     : "0x72dd4b6bd852a3aa172be4d6c5a6dbec588cf131",
    "decimals"    : "18",
    "name"        : "NAGA Coin",
    "ens_address" : "",
    "website"     : "https://www.nagaico.com",
    "logo": {
      "src"       : "",
      "width"     : "",
      "height"    : "",
      "ipfs_hash" : ""
    },
    "support": {
      "email"     : "support@nagaico.com",
      "url"       : ""
    },
    "social": {
      "blog"      : "",
      "chat"      : "",
      "facebook"  : "https://www.facebook.com/thenagaico/",
      "forum"     : "",
      "github"    : "",
      "gitter"    : "",
      "instagram" : "",
      "linkedin"  : "",
      "reddit"    : "",
      "slack"     : "",
      "telegram"  : "",
      "twitter"   : "https://twitter.com/thenagaico",
      "youtube"   : ""
    }
  }`

Comment (SteveMieskoski) [inline tokens/tokens-eth.json]: Please revert removal of the NGC, NAGA Coin entry
`    {
    "symbol"      : "NGC",
    "address"     : "0x72dd4b6bd852a3aa172be4d6c5a6dbec588cf131",
    "decimals"    : "18",
    "name"        : "NAGA Coin",
    "ens_address" : "",
    "website"     : "https://www.nagaico.com",
    "logo": {
      "src"       : "",
      "width"     : "",
      "height"    : "",
      "ipfs_hash" : ""
    },
    "support": {
      "email"     : "support@nagaico.com",
      "url"       : ""
    },
    "social": {
      "blog"      : "",
      "chat"      : "",
      "facebook"  : "https://www.facebook.com/thenagaico/",
      "forum"     : "",
      "github"    : "",
      "gitter"    : "",
      "instagram" : "",
      "linkedin"  : "",
      "reddit"    : "",
      "slack"     : "",
      "telegram"  : "",
      "twitter"   : "https://twitter.com/thenagaico",
      "youtube"   : ""
    }
  }`

**Comment by SteveMieskoski** [comment]
@stephenhodgkiss I made some comments to your commit.  Particularly they relate to both the changing of another token's name and the removal of a token.  Please revert these changes or indicate why they should proceed (with links to blogs, or sites that indicate such) then I will merge in your changes.  Thanks!

**Comment by stephenhodgkiss** [comment]
Very strange Steve, as all I did was to copy the existing TWN symbol lines and amended them for ours. I will take a look.

**Comment by stephenhodgkiss** [comment]
@SteveMieskoski  - okay I found the discrepancies and now the differences state just our new coin.

**Comment by SteveMieskoski** [comment]
@stephenhodgkiss Thanks.

**Pull request merged**